### DISCUSSION TITLE
Domain List is empty

### DISCUSSION THREAD

**Discussion by isanketsaha** [vercel/community#6949] [Help] [answered]
### Summary

I have a Project created (tulip-app) which has the domain added from GoDaddy(added nameserver). So technically the domain has to be managed under Vercel. But I am unable to do so. The team dashboard domain tab has no domain listed, whereas the project has domain added and my website is also up and running under the same domain. 
I am confused and need some guidance. (I basically want to update my email TXT to my domain.)
Attached screenshot for your reference. 

<img width="1364" alt="Screenshot 2024-05-23 at 12 21 36 PM" src="https://github.com/vercel/community/assets/6908935/e242c5fb-8916-487a-8f8e-b1c2af34d881">

<img width="1364" alt="Screenshot 2024-05-23 at 12 22 16 PM" src="https://github.com/vercel/community/assets/6908935/4a91a9d7-01e7-4840-8f2f-0f4cd8f5a007">

### Steps to Reproduce

My project URL - (https://vercel.com/sanketsaha/tulip-app/settings/domains)  have domain listed. 

whereas URL - (https://vercel.com/sanketsaha/~/domains)  don't have any domain listed. 
I have brought the domain from GoDaddy and updated the nameserver so technically my DNS should be managed from vercel.
Please guide me.

**Comment by amyegan** [comment, marked as answer]
Hey @isanketsaha. When you added the domain, it was likely added to your personal account rather than your hobby team. Instead of looking at your team's domains list at [vercel.com/sanketsaha/~/domains](https://vercel.com/sanketsaha/~/domains), please check the domain list for your personal account at [vercel.com/account/domains](https://vercel.com/account/domains)

If you prefer to manage the domain on your hobby team, you can move it using the steps described here: [vercel.com/docs/projects/domains/transfer-your-domain#transfer-a-domain-to-another-vercel-user-or-team](https://vercel.com/docs/projects/domains/transfer-your-domain#transfer-a-domain-to-another-vercel-user-or-team)
